packages/hoppscotch-app/helpers/types/HoppTestResult.ts

```diff
@@ -13,4 +13,5 @@ export type HoppTestResult = {
   tests: HoppTestData[]
   expectResults: HoppTestExpectResult[]
   description: string
+  scriptError?: boolean
```

Comment:
Why is this optional ? Couldn't we just set this to false on error ?

Reply:
I thought of only including it in case of an error. While we can certainly keep this `false` for correct test results. I made it optional to make less changes to existing code.

Reply:
Offlined. Making `scriptError` field non-optional seems a better solution in maintaining long term code consistency. 
Required changes have been made.